src/ansys/fluent/core/solver/flobject.py

```diff
@@ -169,6 +169,15 @@ def to_python_name(fluent_name: str) -> str:
     return name
 
 
+def _get_root_from_child_object(obj):
```

Comment:
This code snippet can be used to extract the root object from any given object.

Reply:
`_root()` could be a private `Base` method.

Reply:
Done, Thank you.